.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,63 @@
+---
+name: New Component
+about: Interested in contributing a new component to Fluent UI React-Native? This template includes necessary information to get started, and steps to completion
+---
+
+<!-- Use this template for new components or new component variants -->
+
+## Component details
+
+### Description
+
+<!-- fill this out -->
+
+### Which product teams/scenarios need this control
+
+<!-- The more teams/scenarios that would use this control the better chance it will get prioritized -->
+
+### Design assets
+
+<!-- Please provide links to redlines or screenshots of intended component design -->
+<!-- Are you internal to Microsoft and your design isn't yet public? Please reach out to us internally via the Fluent UI Apple Teams channel or log a request at https://aka.ms/fluentui-native. -->
```

Comment:
The FluentUI Apple teams channel is too specific. Should apply to all platforms. same with the https://aka.ms/fluentui-native

Reply:
Recommendations on what to change it to?

Reply:
I think it's better to actually have a direct link since Fluent UI Community is pretty vague, but one doesn't exist now so I'm good with starting with this and if/when we create a community teams channel we can update documentation